### PR TITLE
{vis}[intel/2017a] wxPython v3.0.2.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2656.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/bbc2604a770305e48a21108c8c56a25e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2048.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9df6ee1b4d31f97af3ab67d8bf65291e for a full test report.

File: easybuild/easyconfigs/g/GStreamer/GStreamer-0.10.36-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+dependencies = [
+    ('GLib', '2.53.5'),
+    ('GObject-Introspection', '1.53.5', '-Python-2.7.13'),
+    ('zlib', '1.2.11'),
+]
+
+# does not work with Bison 3.x
+builddependencies = [
+    ('Bison', '2.7', '', True),
```
Comment (vanzod) [inline easybuild/easyconfigs/g/GStreamer/GStreamer-0.10.36-intel-2017a.eb]: Why the `dummy` toolchain version?

File: easybuild/easyconfigs/g/GStreamer/GStreamer-0.10.36-intel-2017a.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+]
```
Comment (vanzod) [inline easybuild/easyconfigs/g/GStreamer/GStreamer-0.10.36-intel-2017a.eb]: Configure fails if `flex` is not explicitly added to the builddeps

File: easybuild/easyconfigs/g/GStreamer/GStreamer-0.10.36-intel-2017a.eb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+description = """GStreamer is a library for constructing graphs of media-handling
+ components. The applications it supports range from simple
+ Ogg/Vorbis playback, audio/video streaming to complex audio
+ (mixing) and video (non-linear editing) processing."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://gstreamer.freedesktop.org/src/gstreamer']
+sources = [SOURCELOWER_TAR_XZ]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/g/GStreamer/GStreamer-0.10.36-intel-2017a.eb]: Missing checksum

File: easybuild/easyconfigs/d/DBus/DBus-1.11.20-intel-2017a.eb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+ to one another.  In addition to interprocess communication, D-Bus helps
+ coordinate process lifecycle; it makes it simple and reliable to code
+ a "single instance" application or daemon, and to launch applications
+ and daemons on demand when their services are needed."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://dbus.freedesktop.org/releases/dbus']
+sources = [SOURCELOWER_TAR_GZ]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/d/DBus/DBus-1.11.20-intel-2017a.eb]: Missing checksum

File: easybuild/easyconfigs/d/dbus-glib/dbus-glib-0.108-intel-2017a.eb [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+version = '0.108'
+
+homepage = 'http://dbus.freedesktop.org/doc/dbus-glib'
+description = """D-Bus is a message bus system, a simple way for applications to talk to one another."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://dbus.freedesktop.org/releases/dbus-glib']
+sources = [SOURCE_TAR_GZ]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/d/dbus-glib/dbus-glib-0.108-intel-2017a.eb]: Missing checksum

File: easybuild/easyconfigs/g/GConf/GConf-3.2.6-intel-2017a.eb [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+homepage = 'https://projects.gnome.org/gconf/'
+description = """GConf is a system for storing application preferences.
+ It is intended for user preferences; not configuration
+ of something like Apache, or arbitrary data storage."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://ftp.gnome.org/pub/GNOME/sources/GConf/%(version_major_minor)s/']
+sources = [SOURCE_TAR_XZ]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/g/GConf/GConf-3.2.6-intel-2017a.eb]: Missing checksum

File: easybuild/easyconfigs/g/GST-plugins-base/GST-plugins-base-0.10.36-intel-2017a.eb [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+description = """GStreamer is a library for constructing graphs of media-handling
+ components. The applications it supports range from simple
+ Ogg/Vorbis playback, audio/video streaming to complex audio
+ (mixing) and video (non-linear editing) processing."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://gstreamer.freedesktop.org/src/gst-plugins-base']
+sources = [SOURCELOWER_TAR_XZ]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/g/GST-plugins-base/GST-plugins-base-0.10.36-intel-2017a.eb]: Missing checksum

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/78b626d44dd2f16e66089387231498c7 for a full test report.

**Pull request merged**